### PR TITLE
Added compare tool dir.

### PR DESCRIPTION
**Pull request by jboero**
Added a q&d script to gen a Google Chart HTML and compare gzip and brotli across speed and compression level.  No change to code base.

### CODE REVIEW

**Comment by jboero** [comment]
W00t thanks got my wibbly wobbly brackets sorted!

**Comment by jboero** [comment]
alt text

File: tests/compare/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Brotli Compare
+Basic scripts to test any file comparing Gzip and Brotli across speed and compression strength.  Output is dumped to an HTML result and plotted with Google Charts.  It can be easily extended or changed to compare other compression engines.  Note that default Brotli compression level (MAX) is not tested as it skews the graph for time but it's worth considering manually.
+
+## Usage
+```
+$ git clone 
+$ ./compare.sh [test file]
```
Comment (eustas) [inline tests/compare/README.md]: I believe, the path is a little bit longer =)

Comment (jboero) [reply tests/compare/README.md]: Yeah sorry I developed it at root of a repo I created. Did you try it out anyway? I can correct the readme or play the pseudocode card.

**Comment by jboero** [comment]
This is silly - the Travis failure has nothing to do with this commit.

**Comment by eustas** [comment]
Hello. Thanks for the PR, but I believe "Squash Compression Benchmark" is already a nice tool for comparing the compressors.

**Comment by eustas** [comment]
https://quixdb.github.io/squash-benchmark/#results

**Comment by jboero** [comment]
Oh I just updated this with a Zstd example but I see your comment.  Thanks I'll use that.

image

**Pull request closed without merging**